en/README.md

```diff
@@ -5,7 +5,18 @@
 
 PX4 is the *Professional Autopilot*. Developed by world-class developers from industry and academia, and supported by an active world wide community, it powers all kinds of vehicles from racing and cargo drones through to ground vehicles and submersibles.
 
-This guide contains everything you need to assemble, configure, and safely fly a PX4-based vehicle.
+This guide contains everything you need to assemble, configure, and safely fly a PX4-based vehicle, including:
+
+- [Getting Started](getting_started/README.md) - basic concepts for beginners needed to build and fly an unmanned vehicle using PX4.
+- [Basic Assembly](assembly/README.md) - how to mount flight controller hardware and wire/connect most important peripherals.
+- [Basic Configuration](config/README.md) - how to install PX4 firmware onto the flight controller hardware, select an airframe, configure/calibrate the core sensors that PX4 needs (e.g. compass, GPS, gyro etc.), setup the battery, setup the radio control transmitter and map flight modes to its switches.
+- [Airframe Builds](airframes/README.md) - gives examples of how to setup different flight controller hardware on common airframes/vehicles.

```

Comment:
You could generally remove the intro word. So this might be "Examples of how to ..."